lib/src/lifecycle_module.dart

```diff
@@ -262,6 +263,32 @@ abstract class LifecycleModule extends Object with Disposable {
     return completer.future;
   }
 
+  /// Ensures a given [Disposable] is disposed when the [LifecycleModule] is
+  /// unloaded.
+  @override
+  @protected
```

Comment:
I don't think we should actually make these `@protected` any more since then it technically doesn't implement `DisposableManager`, right? It's weird. I don't really like computers.